frontend/src/utils/python-poet/poet.ts

```diff
@@ -33,24 +33,23 @@ export class Variable implements PythonCode {
 }
 
 interface LiteralOptions {
-  objectAsFieldNames?: boolean;
   removeNull?: boolean;
   removeUndefined?: boolean;
 }
 
 export class Literal implements PythonCode {
   constructor(
-    public value: unknown,
-    public opts: LiteralOptions = {
-      objectAsFieldNames: false,
-      removeNull: false,
-      removeUndefined: true,
-    },
+    public readonly value: unknown,
+    public readonly opts: LiteralOptions = {},
   ) {}
 
+  static from(value: unknown, opts: LiteralOptions = {}): Literal {
+    return new Literal(value, opts);
+  }
+
   toCode(): string {
     const EMPTY_VALUE = "";
-    const { objectAsFieldNames, removeNull, removeUndefined } = this.opts;
+    const { removeNull = false, removeUndefined = true } = this.opts;
```

Comment:
this way properly using adds the defaults. previously we would overwrite other defaults